lib/core/terraform_config/policy.rb

```diff
@@ -0,0 +1,147 @@
+# frozen_string_literal: true
+
+module TerraformConfig
+  # rubocop:disable Metrics/ClassLength
```

Comment:
Nit: This can be moved to the end of line 5. That way we don't have to enable the rule again later, it gets disabled only for the block.